domain/schema/model/sql/0017-machine.sql

```diff
@@ -123,3 +123,16 @@ CREATE TABLE machine_status_data (
     REFERENCES machine (uuid),
     PRIMARY KEY (machine_uuid, "key")
 );
+
+-- machine_removals table is a table which represents machines that are marked
+-- for removal.
+-- Being added to this table means that the machine is marked for removal,
+-- however, we keep a mark_for_removal column to allow for more granularity in
+-- behavior if needed.
+CREATE TABLE machine_removals (
+    machine_uuid TEXT NOT NULL PRIMARY KEY,
+    mark_for_removal BOOLEAN NOT NULL,
```

Comment:
**suggestion**: This is just a thought, what about dropping the bool and keeping a table of only machineUUIDs? Having the machine listed in the table is already a bool. And my opinion is that a separate column will not be used in the future, since the intent of this table is very clearly stated (temporary placeholder for machines that are to be removed). 

If we do this, then we don't even care about the `OR REPLACE` in the insert query, and we can simply ignore the "already exists" error. This operation remains idempotent.

Reply:
Yes. Do this; it's congruent with the current collection.

Reply:
@manadart done, see 72dab1ed1e1401da74f5984e05aff0e3dbaeb7ae